coala_quickstart/generation/SettingsFilling.py

```diff
@@ -32,38 +35,41 @@ def fill_section(section,
     """
     # Retrieve needed settings.
     prel_needed_settings = {}
+
     for bear in bears:
         needed = bear.get_non_optional_settings()
         for key in needed:
             if key in prel_needed_settings:
-                prel_needed_settings[key].append(bear.name)
+                prel_needed_settings[key]["bears"].append(bear.name)
             else:
-                prel_needed_settings[key] = [needed[key][0],
-                                             bear.name]
+                prel_needed_settings[key] = {
+                    "help_text": needed[key][0],
```

Comment:
Agreed. But this is based on existing design which I followed and definitely a topic for a separate issue/PR.